src/dstack/_internal/server/services/backends/configurators/runpod.py

```diff
@@ -0,0 +1,91 @@
+import json
+from typing import List
+
+from dstack._internal.core.backends.base import Backend
+from dstack._internal.core.backends.runpod import RunpodBackend, RunpodConfig, api_client
+from dstack._internal.core.models.backends.base import (
+    BackendType,
+    ConfigElementValue,
+    ConfigMultiElement,
+)
+from dstack._internal.core.models.backends.runpod import (
+    RunpodConfigInfo,
+    RunpodConfigInfoWithCreds,
+    RunpodConfigInfoWithCredsPartial,
+    RunpodConfigValues,
+    RunpodCreds,
+    RunpodStoredConfig,
+)
+from dstack._internal.server.models import BackendModel, ProjectModel
+from dstack._internal.server.services.backends import Configurator
+from dstack._internal.server.services.backends.configurators.base import (
+    raise_invalid_credentials_error,
+)
+
+REGIONS = [
+    "CA-MTL-1",
+    "EU-NL-1",
+    "EU-RO-1",
+    "EU-SE-1",
+    "EUR-IS-1",
+    "EUR-IS-2",
+    "EUR-NO-1",
+    "US-OR-1",
+]
+
+DEFAULT_REGION = "CA-MTL-1"
+
+
+class RunpodConfigurator(Configurator):
+    TYPE: BackendType = BackendType.RUNPOD
+
+    def get_default_configs(self) -> List[RunpodConfigInfoWithCreds]:
+        return []
+
+    def get_config_values(self, config: RunpodConfigInfoWithCredsPartial) -> RunpodConfigValues:
+        config_values = RunpodConfigValues()
+        if config.creds is None:
+            return config_values
+        self._validate_runpod_api_key(config.creds.api_key)
+        config_values.regions = self._get_regions_element(
+            selected=config.regions or [DEFAULT_REGION]
+        )
+        return config_values
+
+    def create_backend(
+        self, project: ProjectModel, config: RunpodConfigInfoWithCreds
+    ) -> BackendModel:
+        if config.regions is None:
+            config.regions = REGIONS
+        return BackendModel(
+            type=self.TYPE.value,
+            config=RunpodStoredConfig(**RunpodConfigInfo.parse_obj(config).dict()).json(),
+            auth=RunpodCreds.parse_obj(config.creds).json(),
+        )
```

Comment:
The `project_id` is required argument. These are the reasons why the model is not created on a fresh installation

```suggestion
        return BackendModel(
            project_id=project.id,
            type=self.TYPE.value,
            config=RunpodStoredConfig(**RunpodConfigInfo.parse_obj(config).dict()).json(),
            auth=RunpodCreds.parse_obj(config.creds).json(),
        )
```